docs/SENDPULSE_INTEGRATION.md

```
@@ -0,0 +1,321 @@
+    "subscription_status": "active",
+    "last_order": "2024-01-15"
+  },
+  "tags": ["vip", "subscription"]
+}
+```
+
+### Webhook Endpoint
+
+#### Incoming Messages
+SendPulse sends messages to:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)